pkg/workloadapi/workload.proto

```diff
@@ -355,3 +362,9 @@ message NamespacedHostname {
   // hostname (ex: gateway.example.com)
   string hostname = 2;
 }
+
+// Extension provides a mechanism to attach arbitrary additional configuration to an object.
+message Extension {
+  string name = 1;
```

Comment:
Does name have any significance? What if two extension are configured with same name? Do we have to prevent it?